.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+AllCops:
+  TargetRubyVersion: 2.3.0
+  Exclude:
+    - 'spec/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+
+Metrics/LineLength:
+  Max: 120
+Style/Documentation:
+  Enabled: false
```

Comment:
Over the years I've convinced myself of the utility of well commented class/module and method definitions.